### PR TITLE
Fix wrong new line

### PR DESCRIPTION
**Pull request by albertvillanova**
Fix wrong new line in docstring.

### CODE REVIEW

**Comment by mihaimaruseac** [comment]
Is there something broken? Afaik this is the markdown syntax for hyperlinks and it should not be split on two lines

**Comment by lamberta** [comment]
Yeah, this change shouldn't matter. Though there is currently a forced line break on the page: https://www.tensorflow.org/api_docs/python/tf/keras/losses/Reduction?version=nightly

If we're stripping `# pylint: disable=line-too-long`, are we adding an extra `<br>`? cc: @yashk2810

**Comment by gbaned** [comment]
@albertvillanova  Any update on this PR? Please. Thanks!

**Comment by gbaned** [comment]
@albertvillanova  Can you please check @lamberta's comments and keep us posted ? Thanks!

**Comment by albertvillanova** [comment]
Hello @gbaned. I thought @lamberta question was addressed to @yashk2810.

Currently, there is a wrong line break in the docs: https://www.tensorflow.org/api_docs/python/tf/keras/losses/Reduction
```
Please see the custom training guide
for more details on this.
```
I have made this pull request to fix it.

Apparently, as @lamberta pointed out, current `# pylint: disable=line-too-long` is creating a line break, so I removed it.

**Comment by albertvillanova** [comment]
In relation to the comment made by @mihaimaruseac, I have not invented split hyperlinks. Indeed they are already used many times in tensorflow project, e.g.:
- https://github.com/tensorflow/tensorflow/blob/bd192e96bc1135cfdd28b169d1e1c3e5e3585c28/tensorflow/python/ops/math_ops.py#L1652
- https://github.com/tensorflow/tensorflow/blob/bd192e96bc1135cfdd28b169d1e1c3e5e3585c28/tensorflow/python/ops/image_ops_impl.py#L1538
- https://github.com/tensorflow/tensorflow/blob/bd192e96bc1135cfdd28b169d1e1c3e5e3585c28/tensorflow/python/ops/image_ops_impl.py#L1538
- https://github.com/tensorflow/tensorflow/blob/bd192e96bc1135cfdd28b169d1e1c3e5e3585c28/tensorflow/python/tpu/tpu_embedding.py#L661
- https://github.com/tensorflow/tensorflow/blob/bd192e96bc1135cfdd28b169d1e1c3e5e3585c28/tensorflow/python/lib/io/tf_record.py#L230

File: tensorflow/python/ops/losses/loss_reduction.py [hunk 1]
```diff
@@ -48,9 +48,9 @@ class ReductionV2(object):
            (1. / global_batch_size)
      ```
 
-  Please see the
-  [custom training guide](https://www.tensorflow.org/tutorials/distribute/custom_training)  # pylint: disable=line-too-long
-  for more details on this.
+  Please see the [custom training guide](
+    https://www.tensorflow.org/tutorials/distribute/custom_training) for more
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/losses/loss_reduction.py:52]:
```suggestion
  Please see the [custom training guide](
  https://www.tensorflow.org/tutorials/distribute/custom_training) for more
```

**Review by mihaimaruseac — COMMENTED** [review]
Thanks. I checked the links you provided and the documentation is not broken on the site.

TIL.

**Review by mihaimaruseac — APPROVED** [review]

**Pull request merged**